twtr/src/integration-test/groovy/twtr/AccountIntegrationSpec.groovy

```diff
@@ -69,13 +65,13 @@ class AccountIntegrationSpec extends Specification {
         account2.save(flush: true)
 
         when:
-        account1.addToFollowing(sus)
-        account2.addToFollowing(sus)
+        sus.addToFollowers(account1)
+        sus.addToFollowers(account2)
 
         then:
         sus.validate()
-        //TBD - add other validation conditions
-
+        sus.followers.count {it} == 2
+        sus.followers.findAll{it -> (it.handle == account1.handle || it.handle == account2.handle)}
```

Comment:
I think this last line should be broken into two separate tests to verify explicitly that each account1 and account2 is following sus.  This doesn't fully verify that as it is written.